src/covizpy/plot_summary.py

```diff
@@ -22,9 +27,68 @@ def plot_summary(
     fun : str, optional
         Aggregation function for val, by default 'sum'
     date_from : str, optional
-        Start date for plot summary, by default '2022-01-01'
+        Start date of the data range with format '%Y-%m-%d'. By default 'None' is used to represent 7 days prior to today's date
     date_to : str, optional
-        End date for plot summary, by default '2022-01-13'
+        End date of data range with format '%Y-%m-%d'. By default 'None' is used to represent today's date
     top_n : int, optional
         Specify number of qualitative values to show, by default 5
     """
+    # Load data
+    df = pd.read_csv("https://covid.ourworldindata.org/data/owid-covid-data.csv")
```

Comment:
Instead of checking below whether the df is present or not, you can check here if the link is broken or not.

`try:
        df = pd.read_csv('https://covid.ourworldindata.org/data/owid-covid-data.csv')
    except:
        return "The link to the data is broken"`

Reply:
I'll also add this exception check to my version in get_data.